SUMMARY.md

```diff
@@ -44,6 +44,7 @@
     * [Constants](reference/interbit/constants.md)
     * [configSelectors](reference/interbit/configSelectors.md)
     * [createChains()](reference/interbit/createChains.md)
```

Comment:
Per your question in standup, it looks like we have an error in this API documentation. `createChains` is not actually a function, but is an object containing two functions: `createChainsFromManifest` and `createChainsFromConfig`(deprecated).

Reply:
When we shake the API tree, issues can sometimes fall out. :smile: I'll file an issue for this.

Reply:
Thanks!